p2p/les.py

```diff
@@ -276,6 +289,9 @@ def send_handshake(self, chain_info: 'ChainInfo') -> None:
             'headHash': chain_info.block_hash,
             'headNum': chain_info.block_number,
             'genesisHash': chain_info.genesis_hash,
+            'serveHeaders': None,
+            'serveChainSince': 0,
+            'txRelay': None,
```

Comment:
Is it correct that this should be set to a truthy value (`1`?) when we have a working transaction pool?

Reply:
Both `txRelay` and `serveHeaders` have no value -- they just need to be present to signal that we do those things. I thought we already had a working tx pool?  If that's not the case I should not include txRelay yet